lib/maths/CTimeSeriesDecompositionDetail.cc

```diff
@@ -292,7 +292,7 @@ const core::TPersistenceTag MEAN_OFFSET_7_11_TAG{"c", "mean_offset"};
 const core::TPersistenceTag RESIDUAL_MOMENTS_7_11_TAG{"d", "residual_moments"};
 const core::TPersistenceTag LARGE_ERROR_FRACTION_7_11_TAG{"e", "large_error_fraction"};
 const core::TPersistenceTag LAST_TEST_TIME_7_11_TAG{"f", "last_test_time"};
-const core::TPersistenceTag LAST_CHANGE_TIME_7_11_TAG{"g", "last_change_time"};
+const core::TPersistenceTag LAST_UNCOMMITTED_CHANGE_TIME_7_11_TAG{"g", "last_uncommitted_change_time"};
```

Comment:
I know it's not directly related to this change, but if you haven't done so already please can you update `api::CAnomalyJob::MODEL_SNAPSHOT_MIN_VERSION` to `7.11.0` on your feature branch so we won't try to load the new format on old nodes.

Reply:
Good point: it's on my list. I was going to bump it at the point I merged to master in case this slipped, but I think that's unlikely now so I can add in this PR.

Reply:
Done in [3617328](https://github.com/elastic/ml-cpp/pull/1550/commits/36173280d512decd1340e2e2e3b7d25771efeabf).